packages/numbers/README.md

```diff
@@ -57,3 +57,87 @@ import { percentage } from '@execonline-inc/numbers';
 
 percentage(42.22) // returns the string "42%"
 ```
+
+# Numeric Predicates
+
+Compare numbers in a declarative way.
+
+## Functions
+
+### `positive`
+
+Check if a value is greater than zero.
+
+```ts
+import { positive } from '@execonline-inc/numbers';
+
+positive(10) // true
+positive(0) // false
+
+```
+
+### `whenPositive`
+
+A function that returns Maybe is available
+
+```ts
+import { whenPositive } from '@execonline-inc/numbers';
+
+const widgetMessage = (value: Maybe<number>) =>
+  value
+    .andThen(whenPositive)
+    .map((count) => `${count} widgets still remain!`)
+    .getOrElseValue("No Remaining widgets!");
+
+widgetMessage(just(7)) // "7 widgets still remain!"
+widgetMessage(just(0)) // "No Remaining widgets!"
+widgetMessage(nothing()) // No Remaining widgets!
+```
+
+### `gt`
+
+Check if a value is greater than.
+
+```ts
+import { gt } from '@execonline-inc/numbers';
+
+const checkForGreaterThan = (value: number): boolean => just(value).map(gt(10)).isJust();
```

Comment:
should I be writing these functions with other options for calling them with non-curried arguments, so we could do `gt(10,7)` for example? I think this is probably a poor example, but I didn't want to write `gt(10)(7)`

Reply:
I wrote my examples curried because I think that is the only scenario you'd really use them. If you ever wrote `gt(10)(7)`, I'd encourage you to just use the operator since that is clearer.